docs/data_ingestion.asciidoc

```diff
@@ -0,0 +1,103 @@
+[[tune-data-ingestion]]
+= Tune Data Ingestion
+
+[partintro]
+--
+APM Server offers a set of <<configuring-howto-apm-server, configuration options>>,
+helping you to adapt Elastic APM according to your needs.
+
+For optimizing your Elastic APM setup read more about how to:
+
+* <<tune-apm-server>>
+* <<tune-es>>
+
+--
+
+[[tune-apm-server]]
+== Tune APM Server
+You can fine tune your setup considering following options:
+
+* <<tune-output-config>>
+* <<increase-queue-size>>
```

Comment:
I know that part of this was there already; but for both `increase-queue-size` and `increase-concurrent-requests` i'd rename titles to "adjust-*" since you can either increase or decrease